src/main/java/uk/gov/pay/webhooks/message/dao/WebhookMessageDao.java

```diff
@@ -5,16 +5,46 @@
 import uk.gov.pay.webhooks.message.dao.entity.WebhookMessageEntity;
 
 import javax.inject.Inject;
+import java.util.List;
 
 public class WebhookMessageDao extends AbstractDAO<WebhookMessageEntity> {
 
+    public static final int WEBHOOK_MESSAGES_PAGE_SIZE = 10;
+
     @Inject
     public WebhookMessageDao(SessionFactory factory) {
         super(factory);
         }
-        
+
     public WebhookMessageEntity create(WebhookMessageEntity webhookMessage) {
         persist(webhookMessage);
         return webhookMessage;
     }
+
+    public WebhookMessageEntity get(String webhookId, String messageId) {
+       return namedTypedQuery(WebhookMessageEntity.MESSAGE_BY_WEBHOOK_ID_AND_MESSAGE_ID)
+               .setParameter("webhookId", webhookId)
+               .setParameter("messageId", messageId)
+               .getSingleResult();
+    }
+
+    public List<WebhookMessageEntity> list(String webhookId, String deliveryStatus, int page) {
+        var query = (deliveryStatus != null) ?
```

Comment:
```suggestion
        var query = deliveryStatus != null ?
```
unnecessary parenthesis.